src/components/atm.button/button.styled.jsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import {FontFamily, FontWeight, FontSize, Color, Border, Padding, ButtonHeight} from '../constants'
+
+const buttonColor = {
+    primary: Color.Primary,
+    secondary: Color.Secondary
+}
+
+export const ButtonStyled = styled.button`
+    width: ${props => props.expansible ? `100%` : `auto` };
+    padding: 0 ${Padding.Medium};
+    line-height: calc(${ButtonHeight});
+    height: ${ButtonHeight};
```

Comment:
nesse caso, uma vez q decidiu fixar a altura, vai dar problema quando o texto do botão for mto grande, ja q n vai ser quebrado em 2 linhas. Isso quer dizer q precisa se preocupar com o overflow, qnd tem um texto longo. (Pode n ter caso assim no design, mas eh legal garantir esses casos de uso mais comuns pelo menos nesses componentes bem genericos)